src/components/TypeBase/TypeBase.js

```diff
@@ -41,10 +42,11 @@ export const TypeBase = ({
   allCaps,
   capitalize,
   color,
-  element,
+  element: Element = 'div',
   subtype,
   typeface,
   weight,
+  elementClasses,
```

Comment:
used in InputLabel:
https://github.com/getethos/ethos-design-system/pull/400/files#diff-cc96ffaf316b77e575c845be3832626befa65dfbda0074f306c96b9f5d903966R54